app.js

```diff
@@ -20,10 +20,16 @@ app.all("/*", (request, response, next) => {
 });
 
 app.use((error, request, response, next) => {
-  if (error.code === "22P02" || error.msg === "404 - No rows found") {
-    response.status(404).send({ msg: "404 - Not found" });
+  if (error.code === "22P02") {
+    response.status(400).send({ msg: "400 - Bad request" });
+  }
+  else if (error.msg) {
+    response.status(404).send({ msg: error.msg || "404 - Not found" })
```

Comment:
Great custom error handling, you could export this out to it's own function later on. See the bottom of these notes - https://notes.northcoders.com/courses/js-back-end/error-handling